lib/chef/resource/cron.rb

```diff
@@ -135,14 +136,7 @@ def weekday(arg = nil)
         )
       end
 
-      def time(arg = nil)
-        set_or_return(
-          :time,
-          arg,
-          :equal_to => Chef::Provider::Cron::SPECIAL_TIME_VALUES
-        )
-      end
-
+      property :time, Symbol, equal_to: Chef::Provider::Cron::SPECIAL_TIME_VALUES
```

Comment:
Technically we didn't restrict this to a symbol, but we expected it to be a symbol in the equal_to so this is the same